util/convert.go

```diff
@@ -31,10 +32,24 @@ func ConvertSignature(s string) ([identity.SignatureSize]byte, error) {
 }
 
 // ConvertStringToken verifies and converts a string token to a proper sized
-// []byte.
+// byte slice. This function accepts both the full length token and token
+// prefixes.
 func ConvertStringToken(token string) ([]byte, error) {
-	if len(token) != pd.TokenSize*2 {
-		return nil, fmt.Errorf("invalid censorship token size")
+	switch {
```

Comment:
I'd add a regex to verify that a token only contains valid characters. It struck me we don't sanitize tokens and we really should,

Reply:
The APIs only accept hex tokens and will error if the token is invalid hex.  Do we need to sanitize further?

Reply:
Done.  Tokens are now validated using the this regexp:
https://github.com/lukebp/politeia/blob/tlog/util/token.go#L26